releases/2022-10-28-releasenotes.md

```diff
@@ -0,0 +1,38 @@
+---
+title: 'We're .COMical and Always Hardening'
+description: 'Vendia is on .com, advanced Azure beta features, expanded success rate for a variety of transaction scenarios, updated SDK defaults, and enhanced experience within GraphQL Explorer'
+date: '2022-10-28'
+draft: false
+tags:
+- Improvements
+- Azure
+- Developer Experience
+---
+
+They're at it again! The Vendia Engineering team has a series of new improvements to continue advancing the experience building on Vendia.  
+
+As always, see our [Docs](https://www.vendia.com/docs/share) for the complete details of each new feature and our [Examples](https://github.com/vendia/examples) for more guidance. 
+Questions? Please [contact us](https://www.vendia.com/contact-us).
+
+# We're COMical 
+While we are quite amusing, we are also officially on [Vendia.com](https://www.vendia.com)! Previously we were nothing-but-net [(Vendia.net)](https://vendia.net/) but now our website is live on Vendia.com. Check it out yourself! Don't worry; development is still on [share.vendia.net](https://share.vendia.net).
+
+# Advancing Azure Beta Feature
+Sharing, without limitation, is our ethos and we continue expanding the surface area on which to share. We've added validations and stabilization for sharing between two azure-backed nodes by resolving errors associated with metadata availability, handling mutations, and cross-cloud file sharing.
+
+# Ensuring Transaction Success
+We continue to make improvements to the speed of [consensus](https://www.vendia.com/docs/share/terms-and-definitions#consensus) and success rate of transactions by reducing scenarios in which errors might display. This release focused on resolving errors due to empty versions of data, complex access rules (ACLs), duplicate attribute names exist in different nested paths, and various uni deployments. We also removed any restrictions caused by WAF API limits.
+
+# Keeping the SDK Current
```

Comment:
Example code here would be good.  Both the generated code (where the default is applied) and the approach to override (if the previous default of `Node_Ledgered` is still desired).

Reply:
added!